storybook/.storybook/preview.ts

```diff
@@ -5,6 +5,7 @@ const preview: Preview = {
     html: {
       root: "#root-inner", // default: #root
       removeEmptyComments: true, // default: false
+      removeComments: /\?lit\$.*\$/,
```

Comment:
😲 is het zo makkelijk?

Ik lees de documentatie nu en ik wist helemaal niet dat dit bestond. Fantastisch! Nog overwogen om 'm op `true` te zetten?

Reply:
ja, maar ik weet niet zeker of er een situatie is waar toch wel een keer een comment nodig is. Met dit hou je die mogelijkheid wel open. Aan de andere kant kunnen er zo ook weer andere comments insluipen.

Reply:
Ja precies, goede afweging

Reply:
Heb hem wel iets simpeler gemaakt, want kwam nog een `<!--?-->` tegen.